property-services/src/main/resources/db/migration/main/V20191121145421__egpt_assessment_tables_ddl.sql

```diff
@@ -0,0 +1,99 @@
+DROP TABLE IF EXISTS  eg_pt_unit;
+DROP TABLE IF EXISTS eg_pt_assessments;
+DROP TABLE IF EXISTS  eg_pt_unit_audits;
+DROP TABLE IF EXISTS eg_pt_assessments_audits;
+
+
+CREATE TABLE eg_pt_assessments (
+
+  id character varying(256) NOT NULL,
+  financialyear character varying(256) NOT NULL,
+  tenantId character varying(256) NOT NULL,
+  assessmentNumber character varying(64) NOT NULL,
+  status character varying(64) NOT NULL,
```

Comment:
Do we not have search by status or other fields ? We'll have to indices on all searchable fields

Reply:
right now we don't, most searches happen on assessmentNumber and propertyId.

Reply:
If it's allowed on the controller, please replicate the same on the DB. Adding additional indices on sparingly used fields is fine.